source/blog/2020-07-22-release-notes.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+title: July 2020 Release Notes - Version 20.7.0
+published: true
+post_author:
+  display_name: Ruben Oanta
+  twitter: rubenoanta
+tags: Releases, Finagle, Finatra, Scrooge, TwitterServer, Util
+---
+
+Hi folks! Our July release is here – enjoy!
+
+### [Finagle](https://github.com/twitter/finagle/)
+
+### Breaking API Changes
+
+-   finagle-core: Correct the spelling of Tracing.recordClientSendFrargmet() to
+    Tracing.recordClientSendFragment() [30726c01](https://github.com/twitter/finagle/commit/30726c014efe95760e9daff00e1813a412cd1bbe)
+-   finagle-redis: Use StrictKeyCommand for XDEL [d174f9d6](https://github.com/twitter/finagle/commit/d174f9d603fa28122e3cdcc1d39563790bcbe98e)
+-   finagle-toggle: Toggle.isDefinedAt(i: Int) has become Toggle.isDefined. Additionally, a new method Toggle.isUndefined has been added. [3bb78ce8](https://github.com/twitter/finagle/commit/3bb78ce8dd10e816f18544143cbd5081a2f13721)
+-   finagle-zipkin-scribe: The scribe.thrift file was moved to finagle-thrift/src/main/thrift under a new
+    namespace. com.twitter.finagle.thrift.scribe.(thrift|thriftscala) [01a20b79](https://github.com/twitter/finagle/commit/01a20b798726550a670b890aa630e4ddd657bddd)
+
+### Bug Fixes
+
+-   finagle-zipkin-scribe: The scribe client should be configured using the NullTracer. Otherwise, spans
+    produced by the client stack will be sampled at initialSampleRate. [0446dd3e](https://github.com/twitter/finagle/commit/0446dd3e5ea519cac15cec990f919113c5a2d503)
+
+### [Finatra](https://github.com/twitter/finatra/)
```

Comment:
This isn't all of the Finatra changelog, this is only the added section, is missing "Changed" and "Fixed"

Reply:
Oof, looks like I missed the rest in the releases too! Fixing now.

Thanks!